package.json

```diff
@@ -6,8 +6,10 @@
         "@commitlint/config-conventional": "8.3.4",
         "@commitlint/travis-cli": "8.3.5",
         "@sindresorhus/slugify": "0.11.0",
+        "@testing-library/cypress": "^7.0.4",
         "aws-sdk": "2.576.0",
-        "axios": "0.19.0",
+        "axios": "^0.21.1",
```

Comment:
What do we need axios for?

Reply:
i guess it got updated when i did a `npm install`